ts/components/wallet/PaymentMethodsList.tsx

```diff
@@ -48,13 +55,15 @@ const paymentMethods: ReadonlyArray<IPaymentMethod> = [
     navigateTo: undefined, // TODO: add route when destination is available @https://www.pivotaltracker.com/story/show/157588719
     name: I18n.t("wallet.methods.bank.name"),
     maxFee: I18n.t("wallet.methods.bank.maxFee"),
-    icon: "io-48-bank"
+    icon: "io-48-bank",
+    onPress: unavailableAlert
```

Comment:
not sure why we show these methods at all? I think it's annoying clicking on something and getting an alert saying it's not available.

I would go for either:

* not showing them altogether
* dimming them out (e.g. rendering with low opacity or in a light gray color)

Reply:
I think this discussion already came up in a previous product Q&A, and the decision was to have this behavior for the time being IIRC